AzureCalling/Views/ParticipantView.swift

```diff
@@ -105,9 +108,44 @@ class ParticipantView: UIView {
         }
 
         placeholderImage.isHidden = isDisplayVideo
+        switchCameraButton.isHidden = !isDisplayVideo
         view.isHidden = !isDisplayVideo
     }
 
+    func updateCameraSwitch(isOneOnOne: Bool) {
+        // Skip update if display mode is the same
+        guard detachMode != isOneOnOne else {
```

Comment:
Could `detachMode` and `isOneOnOne` be the same name?

Reply:
Sure. I could also use an alias for the parameter: `updateCameraSwitch(isOneOnOne detachMode: Bool)`